alot/defaults/alot.rc.spec

```diff
@@ -327,6 +327,12 @@ thread_focus_linewise = boolean(default=True)
         # list of tags to automatically add to draft messages
         draft_tags = force_list(default=list('draft'))
 
+        # list of tags to automatically add to replied messages
+        replied_tags = force_list(default=list('replied'))
+
+        # list of tags to automatically add to passed messages
+        passed_tags = force_list(default=list('passed'))
```

Comment:
this is going to parse down to `force_list(default=['p', 'a', 's', 's', 'e', 'd'])`, that seems off to me. I think this should be `force_list(default=['passed'])`

Reply:
Again, I followed the existing style for sent and draft; but you are right so obviously that I should have noticed, sorry. (I didn't notice the effect because of explicit config.)

Being new here: Do you prefer a clean-up patch for sent and draft as a new 1/3 for this little series, or fixing it along with 1/2?